bluer-tools/src/blumon.rs

```diff
@@ -13,9 +13,92 @@ use std::{
     io::stdout,
     iter,
     time::{Duration, Instant},
+    error::Error,
 };
+
 use tokio::time::sleep;
 
+use chrono::{DateTime, Utc};
+use serde::{Serialize, Deserialize};
+use serde_jsonlines::append_json_lines; //https://jsonlines.org/
```

Comment:
```suggestion
use serde_jsonlines::append_json_lines; 
```

Reply:
I think some comment would be useful here about jsonlines and where to find information on it since people may not know the best place to look